README.md

```diff
@@ -20,7 +20,7 @@
 <h2 align="center">Install</h2>
 
 ```bash
-npm i -D uglifyjs-webpack-plugin@beta
+npm i -D uglifyjs-webpack-plugin@latest
```

Comment:
Please just remove the appendix altogether :)

```js
npm i -D uglifyjs-webpack-plugin
```

Reply:
done!